### PR TITLE
skip hosts to ignore

### PR DESCRIPTION
**Pull request by Mr-Pi**

### CODE REVIEW

File: scripts/reverseproxy-facts.py
Comment (SlothOfAnarchy) [inline scripts/reverseproxy-facts.py]: I don't think so

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]
What about the `ignore_domains` option?

File: scripts/reverseproxy-facts.py
Comment (SlothOfAnarchy) [inline scripts/reverseproxy-facts.py]: It seems like there would be an error if there is no key `ignore_hosts`, yet the option is not declared mandatory in readme. How about `if 'ignore_hosts' in facts and host in facts['ignore_hosts']:`?

File: scripts/reverseproxy-facts.py
Comment (dasJ) [reply scripts/reverseproxy-facts.py]: Yep, he forgot `et`

Comment (dasJ) [reply scripts/reverseproxy-facts.py]: Also, the modeline is compatible with vi, sou you can use `vi:` instead of `vim:`

Comment (dasJ) [reply scripts/reverseproxy-facts.py]: And shouldn't there be a trailing colon?

**Comment by Mr-Pi** [comment]
Please rereview

**Review by SlothOfAnarchy — DISMISSED** [review]
Ignoring domains is still missing.
I'm sorry, I think I wasn't clear. I wanted you to remove the modeline, it is unnecessary.

**Comment by Mr-Pi** [comment]
You are right, I will fix the ignore_domains option. But I will not remove the vi modeline, we used a diffrent python typo than the vim default. Since python is a programming language where indention is important a proper modeline is common sense at the vim and python community. For reverence see https://wiki.python.org/moin/Vim .

**Comment by SlothOfAnarchy** [comment]
Too bad your editor has strange defaults. When I originally pushed the script, I formatted it using the official python style guide [PEP 8](https://www.python.org/dev/peps/pep-0008/). This change and comment in general is unnecessary, I won't merge it.

**Comment by Mr-Pi** [comment]
Please review

**Review by SlothOfAnarchy — DISMISSED** [review]
Looks fine, now it's just the unnecessarily introduced comment.
I do think it's a valid reason. It's not needed for the script and I'm sure if I were to write an editor that needs bloat like that you wouldn't want me to add comments to every document because of it.

**Comment by Mr-Pi** [comment]
A useful addition to Python source files is this comment:
`# vim: tabstop=8 expandtab shiftwidth=4 softtabstop=4`

**Comment by SlothOfAnarchy** [comment]
Feel free to add this in your local version. My editor indents python code just fine without it.

**Comment by dasJ** [comment]
Unfortunately, I don't know Python :/

**Review by dasJ — APPROVED** [review]
Looks good to me - Please sort out that modeline issue before merging/removing the line

**Pull request merged**
